docs/zh_cn/dataset.md

```diff
@@ -243,31 +243,52 @@ python ./tools/convert_datasets/tao/merge_coco_with_lvis.py --lvis ./data/lvis/a
 python ./tools/convert_datasets/tao/tao2coco.py -i ./data/tao/annotations --filter-classes
 
 # LaSOT
+# 有两种预处理数据集的选择:
+# 选择1: 使用离线的coco格式的标注。相应地，在训练中会使用 `SOTTrainDataset` 类, 在测试中会使用 `SOTTestDataset` 类
 python ./tools/convert_datasets/lasot/lasot2coco.py -i ./data/lasot/LaSOTBenchmark -o ./data/lasot/annotations
+# 选择2: 使用标注在线解析方法。相应地，在训练和测试中都使用 `LaSOTDataset` 类。
```

Comment:
forgot delete it

Reply:
I find we don't need to use the  coco-format  dataset for SOT dataest any more,  because I have replaced the coco-format paring with online paring in test dataset configs of SiamRPN++ and the train set of SiamRPN++ doesn't involve the SOT datasets.